R/utils-classes.R

```diff
@@ -288,18 +286,19 @@ collect.bcdc_promise <- function(x, ...){
 show_query.bcdc_promise <- function(x, ...) {
 
   x$query_list$CQL_FILTER <- finalize_cql(x$query_list$CQL_FILTER)
+  cql_filter <- x$query_list$CQL_FILTER
+  x$query_list$CQL_FILTER <- NULL
 
-  url <-  x$cli$url_fetch(query = x$query_list)
-
-  url <- url_format(url)
+  url <-  x$cli$url_fetch()
 
   cat_line("<url>")
   cat_line(url)
   cat_line()
-  cat_line("<SQL>")
-  cat_line(x$query_list$CQL_FILTER)
+  cat_line("<body>")
+  cat_line(glue::glue("   {names(x$query_list)}: {x$query_list}"))
+  cat_line(glue::glue("   CQL_FILTER: {substr(cql_filter, 1, 40)}..."))
 
-  invisible(x$cli$url_fetch(query = x$query_list))
+  invisible(TRUE)
```

Comment:
Do we no longer want to return anything here? I guess it's a bit trickier with `POST`, since `GET` was just a url?

Reply:
I'm glad you caught this. Yeah the url is almost meaningless at this point. Just an endpoint to supply the body. So to me it doesn't make much sense anymore.